### PR TITLE
adds shippable_jdk executable

### PR DESCRIPTION
**Pull request by kanthchandra**
fixes https://github.com/dry-dock/u14all/issues/62

### CODE REVIEW

File: services/shippable_jdk [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    export PATH="$PATH:/usr/lib/jvm/java-8-oracle/bin";
+    java_path="/usr/lib/jvm/java-8-oracle/jre/bin/java";
+    javac_path="/usr/lib/jvm/java-8-oracle/bin/javac";
+  elif [ "$JDK_VERSION" == "oraclejdk9" ]; then
+    export JAVA_HOME="/usr/lib/jvm/java-9-oracle";
+    export PATH="$PATH:/usr/lib/jvm/java-9-oracle/bin";
+    java_path="/usr/lib/jvm/java-9-oracle/bin/java";
+    javac_path="/usr/lib/jvm/java-9-oracle/bin/javac";
+  else
+    export SHIPPABLE_JDK_VERSION="oraclejdk8"
```
Comment (rageshkrishna) [inline services/shippable_jdk]: We should not fall back to a default. This case should print an error message that tells the user that the requested JDK version is unknown and exit with a non-zero code.

**Comment by kanthchandra** [comment]
I'm closing this PR for now. I'll update and reopen this PR tomorrow

File: services/shippable_jdk [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    export JAVA_HOME="/usr/lib/jvm/java-8-oracle";
+    export PATH="$PATH:/usr/lib/jvm/java-8-oracle/bin";
+    java_path="/usr/lib/jvm/java-8-oracle/jre/bin/java";
+    javac_path="/usr/lib/jvm/java-8-oracle/bin/javac";
+  elif [ "$JDK_VERSION" == "oraclejdk9" ]; then
+    export JAVA_HOME="/usr/lib/jvm/java-9-oracle";
+    export PATH="$PATH:/usr/lib/jvm/java-9-oracle/bin";
+    java_path="/usr/lib/jvm/java-9-oracle/bin/java";
+    javac_path="/usr/lib/jvm/java-9-oracle/bin/javac";
+  else
```
Comment (kanthchandra) [inline services/shippable_jdk]: earlier versions of jdk are not there from v5.7.3 onwards only these 5 are there. So checking only for these 5.

**Comment by rageshkrishna** [comment]
This should go to `u14`. We should also have a check for each version to verify that the file exists where we expect it to be. If it's not present, error and exit with a code.

**Comment by kanthchandra** [comment]
Closing this based on the above comment and moved this file to u14 instead of u14all.

**Pull request closed without merging**